CHANGE_LOG

```diff
@@ -1,3 +1,309 @@
+Version 0.56.0 (TBC June, 2022)
+-------------------------------
+
+This release continues to add new features, bug fixes and stability improvements
+to Numba. Please note that this will be the last release that has support for
+Python 3.7 as the next release series (Numba 0.57) will support Python 3.11!
+
+Python language support enhancements:
+
+* Previously missing support for large in-line dictionaries and internal calls
+  to functions with large numbers of keyword arguments in Python 3.10 has been
+  added.
+* ``operator.mul`` now works for ``list`` s.
+* Literal slices, e.g. ``slice(1, 10, 2)`` can be returned from ``nopython``
+  mode functions.
+* The ``len`` function now works on ``dict_keys``, ``dict_values`` and
+  ``dict_items`` .
+* Numba's ``set`` implementation now supports reference counted items e.g.
+  strings.
+
+Numba specific feature enhancements:
+
+* The experimental ``jitclass`` feature gains support for a large number of
+  ``builtin`` methods e.g. declaring ``__hash__`` or ``__getitem__`` for a
+  ``jitclass`` type.
+* It's now possible to use ``@vectorize`` on an already ``@jit`` family
+  decorated function.
+* Name mangling has been updated to emit compiled function names that exactly
+  match the function name in Python. This means debuggers, like GDB, can be set
+  to break directly on Python function names.
+* A GDB "pretty printing" support module has been added, when loaded into GDB
+  Numba's internal representations of Python/NumPy types are rendered inside GDB
+  as they would be in Python.
+* An experimental option is added to the ``@jit`` family decorators to entirely
+  turn off LLVM's optimisation passes for a given function (see
+  ``_dbg_optnone`` kwarg in the ``@jit`` decorator family).
+* A new environment variable is added ``NUMBA_EXTEND_VARIABLE_LIFETIMES``, which
+  if set will extend the lifetime of variables to the end of their basic block,
+  this to permit a debugging experience in GDB similar to that found in compiled
+  C/C++/Fortran code.
+
+NumPy features/enhancements:
+
+* Initial support for passing, using and returning ``numpy.random.Generator``
+  instances has been added, this includes some commonly used distributions.
```

Comment:
Since #8038 didn't make it, do we need to adapt this line?

Reply:
Have changed it, f046462.